asapdiscovery-cli/asapdiscovery/cli/cli_args.py

```diff
@@ -166,3 +167,30 @@ def cache_type(func):
         multiple=True,
         help="The type of cache to use, can be 'JSON' or 'DesignUnit', an be specified multiple times to use cache",
     )(func)
+
+
+def md(func):
```

Comment:
does it make sense to add an `md_args` method option like you have for the `dask_args`?

Reply:
great idea